package.json

```diff
@@ -31,8 +31,8 @@
     "postversion": "git push && git push --tags"
   },
   "dependencies": {
-    "bootstrap": "~3.3.7",
-    "jquery": "^2.1.3",
+    "bootstrap": "^3.4.1",
+    "jquery": "^3.4.1",
```

Comment:
i see we are using jquery 2 in managers app, is it an issue?

Reply:
Couldn't find any issue, but I can't find any real usage of `jQuery` in the codebase 🤷‍♂

Reply:
`yarn why jQuery`
> `[1/4] 🤔  Why do we have the module "jQuery"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
error We couldn't find a match!`

Reply:
```sh
$ yarn why jquery
yarn why v1.21.1
[1/4] 🤔  Why do we have the module "jquery"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "jquery@2.2.4"
info Has been hoisted to "jquery"
info This module exists because it's specified in "dependencies".
✨  Done in 0.47s.
```

Reply:
jQuery seems to be used but not imported.

see: https://github.com/ovh-ux/ng-ovh-responsive-tabs/blob/master/src/directive.js#L29-L30

Reply:
https://github.com/jquery/jquery-migrate/#README < I'll use this to make sure nothing breaks